src/mondo_robot_template.py

```diff
@@ -0,0 +1,75 @@
+"""Medgen->Mondo robot template
+
+Create a robot template to be used by Mondo to add MedGen xrefs curated by MedGen.
+
+See also:
+- PR: https://github.com/monarch-initiative/medgen/pull/9
+- Used here: https://github.com/monarch-initiative/mondo/pull/6560
+"""
+from argparse import ArgumentParser
+from copy import copy
+from pathlib import Path
+from typing import Dict, List
+
+import pandas as pd
+
+SRC_DIR = Path(__file__).parent
+PROJECT_DIR = SRC_DIR.parent
+FTP_DIR = PROJECT_DIR / "ftp.ncbi.nlm.nih.gov" / "pub" / "medgen"
+INPUT_FILE = str(FTP_DIR / "MedGenIDMappings.txt")
+OUTPUT_FILE = str(PROJECT_DIR / "ROBOT_addMedGen_fromIngest.tsv")
+
+
+def _prefixed_id_rows_from_common_df(source_df: pd.DataFrame, mondo_col='mondo_id', xref_col='xref_id') -> List[Dict]:
```

Comment:
Lets do a walk thru of this in our next call?

Reply:
Sure. It's largely the same logic as in the Perl code that we went over and edited on a past call. Just chooses the correct namespace(s) for a given term.